dashboard/app/controllers/scripts_controller.rb

```diff
@@ -163,6 +163,30 @@ def standards
     @unit_summary = @script.summarize_for_rollup(@current_user)
   end
 
+  def get_rollup_resources
+    script = Script.get_from_cache(params[:id])
+    course_version = script.get_course_version
+    return unless course_version
```

Comment:
Would we want to give an error message for this? It may not be necessary if we fix this issue where we just don't show the teacher and student resources unless you set the course version (https://codedotorg.atlassian.net/browse/PLAT-995)

Reply:
I'll definitely fix the issue with showing the resource editing without a course version but I think it's worth returning something useful here. Updated with an error code and message